packaging/rpm/microshift.spec

```diff
@@ -61,7 +61,16 @@ Requires: sos
 %{?systemd_requires}
 
 %description
-The MicroShift package provides an OpenShift Kubernetes distribution optimized for small form factor and edge computing.
+The microshift package provides an OpenShift Kubernetes distribution optimized for small form factor and edge computing.
```

Comment:
Should we add a dependency on the new RPM so that it's automatically deployed when "dnf install -y microshift" command is run?

Reply:
Well, the problem is this information isn't needed on the machine where MicroShift is run, it is needed on the machine where osbuild-composer blueprints are updated (which may not even be an RPM-based machine, so this solution is probably just temporary). So I don't think adding this a required dependency (even if small) is relevant.

Reply:
I would like to see a document on the usage scenarios before I make up my mind about this.

Reply:
The contents of the files going into this new RPM are also compiled into the microshift binary like the other assets, so microshift does not need the files to be installed. It would not be wrong to install them on the host running microshift, but nothing would necessarily consume them.

Reply:
Still, a document is missing in this PR. We need to document in GitHub what a dev can do with this RPM. A mostly copy-paste from the proposal doc?

Reply:
Yes, we should not add a dependency, but documentation is always good.